device/celestica/x86_64-cel_midstone-r0/Midstone-200i/port_config.ini

```diff
@@ -0,0 +1,65 @@
+# name              lanes                   speed       index   mtu
+Ethernet0           249,250,251,252         100000      0       9126
```

Comment:
index match the front panel port number.

Reply:
can you elaborate?

Reply:
https://github.com/Azure/SONiC/wiki/Porting-Guide 

section port_config.ini

Reply:
I went over the doc. Do you mean index should start from 1 instead of 0?

Reply:
@lguohan - does having Index start at 1 instead of 0 have impact on the platform?  We wanted to keep behavior same by matching indexing as if index column wasn't there (default starting at 0)

Reply:
this column is used for show user which front panel port number it is. therefore it should match front panel port number.

Reply:
The show interface transceiver eeprom/presence commands are adversely impacted if we switch to 1 base.  There seems to be a degree of hardcoding to start at 0 on both ODM and SONiC side.  For most folks we worked with, they use Alias column to show front panel mapping / breakout.

Reply:
which part is hardcoded? you can check other platform is using index starting from 1.

https://github.com/Azure/sonic-buildimage/blob/master/device/dell/x86_64-dell_z9100_c2538-r0/Force10-Z9100-C32/port_config.ini

Reply:
updated